exchanges/binanceus/binanceus_test.go

```diff
@@ -506,31 +505,31 @@ func TestGetSinglePriceData(t *testing.T) {
 
 func TestGetAveragePrice(t *testing.T) {
 	t.Parallel()
-	_, err := bi.GetAveragePrice(context.Background(), currency.NewPair(currency.BTC, currency.USDT))
+	_, err := bi.GetAveragePrice(t.Context(), currency.NewPair(currency.BTC, currency.USDT))
```

Comment:
🦍 🚫 I was going to suggest this, but I'm saying it so often I think I'll just open a PR once this merges that fixes them all at once.
```suggestion
	_, err := bi.GetAveragePrice(t.Context(), currency.NewBTCUSDT())
```

Reply:
```
perl -pi -E 's{currency.NewPair\(currency.BTC, currency.(USDT?)\)}{currency.NewBTC$1()}g' **/*.go
```